sample-lnd.conf

```diff
@@ -219,6 +219,10 @@
 ; The target location of the channel backup file.
 ; backupfilepath=~/.lnd/data/chain/bitcoin/simnet/channel.backup
 
+; The maximum capacity of the block cache in bytes.
```

Comment:
Perhaps add some explanation re what the impact of increasing this value is so that it's easier to decide whether it's relevant to you?